sdk/keyvault/keyvault-common/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@azure/keyvault-common",
+  "sideEffects": false,
+  "private": true,
```

Comment:
Wouldn't this library need to ship? If so why it is marked private?

Reply:
This is not a library, it's a shared folder that gets built and shipped through the other packages: keyvault-keys, keyvault-secrets and keyvault-certificates.

The package.json is there so that VSCode can pick up the dependencies.